exchanges/bitstamp/bitstamp.go

```diff
@@ -86,6 +83,40 @@ func (b *Bitstamp) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder)
 	return fee, nil
 }
 
+// GetTradingFee returns a trading fee based on a currency
+func (b *Bitstamp) getTradingFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (float64, error) {
+	fees, err := b.GetAccountTradingFee(ctx, feeBuilder.Pair)
+	if err != nil {
+		return 0, err
+	}
+	if len(fees) != 1 {
+		return 0, errors.New("did not receive exactly one fee from api")
+	}
+	fee := fees[0].Taker
+	if feeBuilder.IsMaker {
+		fee = fees[0].Maker
+	}
+	return fee / 100 * feeBuilder.PurchasePrice * feeBuilder.Amount, nil
+}
+
+// GetAccountTradingFee returns a TradingFee for a pair
+func (b *Bitstamp) GetAccountTradingFee(ctx context.Context, pair currency.Pair) ([]MakerTakerFees, error) {
```

Comment:
This endpoint does not work. Please add a test
```
    bitstamp_test.go:874: json: cannot unmarshal object into Go struct field TradingFees.Fees of type []bitstamp.MakerTakerFees
    bitstamp_test.go:877: []
```

Reply:
Noted, working on it